### DISCUSSION TITLE
Jellyseer on Windows

### DISCUSSION THREAD

**Discussion by LazeX7** [seerr-team/seerr#780] [Q&A] [answered]
I'm a beginner so bear with me!

I have installed jellyseer in a docker container and that works very well. But I want to make Jellyseer accessible externally, I managed to do this with Jellyfin and with duckdns. My Jellyfin runs on a Windows machine, but I failed to understand how to install Jellyseer on Windows.

After many tries I failed to get my current installation externally accessible because. I did a portfowarding and with the external IP and port number it works. But I cant seem to figured out DuckDNS with Caddy in docker.

This is the reason I want to try Jellyseer on Windows, but is it just me or are the instruction for installing it on Windows ver minimalistic and not helpful?

Windows
Pre-requisites:

Nodejs
[Yarn]
Download/git clone the source code from the github (Either develop branch or main for stable)
npm i -g win-node-env
set CYPRESS_INSTALL_BINARY=0
yarn install --frozen-lockfile --network-timeout 1000000
yarn run build
yarn start

I have installed both applications and downloaded the source, but I failed to understand where I need to put the source after downloading it.

**Comment by LazeX7** [comment]
I get the follow error

![image](https://github.com/Fallenbagel/jellyseerr/assets/14087562/146b505a-01ee-4d68-abdc-212bc8835079)

**Comment by gauthier-th** [comment]
From the 1.7 release note:
> Before you yarn build please downgrade your typeorm version by yarn add typeorm@0.3.11
typeorm/typeorm#9860 (comment)

**Reply by LazeX7** [reply]
Any advise on how to do that?

**Reply by gauthier-th** [reply, marked as answer]
Before doing "yarn run build" run the command "yarn add typeorm@0.3.11" and then "yarn build" and "yarn start"

**Reply by LazeX7** [reply]
Omg! you're the best, thanks!
